addons/tmf_loadouts/loadouts/us_army_1970.hpp

```diff
@@ -0,0 +1,533 @@
+
+/* Dependencies:
+		HLC
+		USM
+		CUP
+		CFP
+		ACE
+		FIR
+*/
+
+//		Created by: AChesheireCat
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {"usm_bdu_odg"};
+	vest[] = {"usm_vest_LBE_rm"};
+	backpack[] = {};
+	headgear[] = {};
+	goggles[] = {};
+	hmd[] = {};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {};
+	scope[] = {};
+	bipod[] = {};
+	attachment[] = {};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {};
+	sidearmAttachments[] = {};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = {
+		 MEDICAL_R ,
+		"ACE_Flashlight_MX991",
+		"ACE_EntrenchingTool"
+	};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		"CUP_H_USArmy_Helmet_M1_Vine",
+		"CUP_H_USArmy_Helmet_M1_Olive",
+		"CUP_H_USArmy_Helmet_M1_plain_Olive",
+		"CUP_H_USArmy_Helmet_M1_plain_Vine",
+		"CUP_H_USArmy_Helmet_M1_btp"
+	};
+	primaryWeapon[] = {"CUP_arifle_M16A1"};
+	magazines[] = {
+		LIST_9("CUP_20Rnd_556x45_Stanag"),
+		LIST_2("CUP_20Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {"usm_vest_LBE_gr"};
+	primaryWeapon[] = {"CUP_glaunch_M79"};
+	sidearmWeapon[] = {"CUP_hgun_Colt1911"};
+	magazines[] += {
+		LIST_20("CUP_1Rnd_HE_M203"),
+		LIST_11("CUP_1Rnd_HE_M203"),
+		LIST_2("CUP_1Rnd_Smoke_M203"),
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_7("CUP_7Rnd_45ACP_1911")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"usm_pack_m5_medic"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"usm_pack_alice_medic"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_Mac10"};
+	magazines[] = {
+		LIST_8("CUP_30Rnd_45ACP_MAC10_M"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
```

Comment:
The M79 was much more common during the Vietnam War while the M203 really only came into proliferation during the latter years. I wanted to try to represent this by giving the much more common FTL role the M79 while the SL got the M16A1 M203. I also feel as though it'd create an interesting gameplay dynamic wherein the FTL is a dedicated grenadier role and can supplement the fireteams firepower much more than if they were simply a rifleman that happened to have a GL.

Also, because the FTL can't carry both an M16A1 and the M79 (which would be the ideal solution).